src/lib.rs

```diff
@@ -344,6 +327,19 @@ mod tests {
         simulate_signing(offline_sign, b"ZenGo");
     }
 
+    #[test]
+    fn test_remove_sign_rotate_sign() {
```

Comment:
I suggest to add here a line for description (or instead in the readme file) saying that  removing a party is simply done by not broadcasting encrypted messages to this party. If enough parties decided not to broadcast for a specific party index - this party will not be able to reconstruct a rotated key

Reply:
👍🏼